packages/terra-section-header/docs/README.md

```diff
@@ -0,0 +1,37 @@
+# Terra Section Header
+
+Section Header presentational component that provides an onClick callback and animatable status icon.
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-section-header`
+
+## Usage
+
+```jsx
+import React from 'react';
+import SectionHeader from 'terra-section-header';
+
+<SectionHeader
+  title="Section Header Example"
+  isAnimated
```

Comment:
The `isAnimated` prop can be removed here since we scoped it out.

Reply:
Done aa42014004dd17cacbdf78f5176b2a74dcfbff6a